wrappers/python/tests/layer/test_llama_attention.py

```diff
@@ -115,21 +125,19 @@ def generate_inputs(dtype: str, params: LlamaAttentionTestParams, bias: bool):
     x_value.from_array(x_nntile)
     x_torch = torch.Tensor(x_nntile.T)
 
-    pos_ids = rng.integers(
+    pos_ids = rng.integers(0,
```

Comment:
This 0 makes no sense. You meant a pair of `low` and `high` values. But If `high` value is not provided, then random numbers are generated in range from 0 to `low`-1.